<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="PostgreSQL - postgres@localhost" uuid="02e060d2-50c1-4ccc-943c-53acf4d043c8">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://localhost:5432/postgres</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="f057330f-7608-42b7-851c-bfe90054ef35">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/test_auth/settings.py</remarks>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://127.0.0.1:5432/calculacusto</jdbc-url>
    </data-source>
  </component>
</project>